src/index.tsx

```diff
@@ -78,7 +78,8 @@ export const useColors = (): [
  */
 export const useGitHubRepo = (
   username: string,
-  repository: string
+  repository: string,
+  useDarkTheme: boolean
```

Comment:
Let's rename this to "dark" and use it as an optional flag.

```suggestion
  dark?: boolean
```